query_optimizer/resolver/Resolver.cpp

```diff
@@ -2670,7 +2681,8 @@ E::ScalarPtr Resolver::resolveFunctionCall(
   }
 
   // Make sure that the aggregate can apply to the specified argument(s).
-  if (!aggregate->canApplyToTypes(argument_types)) {
+  if ((aggregate != nullptr && !aggregate->canApplyToTypes(argument_types))
```

Comment:
Code style for the bool expressions:

https://google.github.io/styleguide/cppguide.html#Boolean_Expressions